notebooks/MAST/K2/K2_FFI/beginner_how_to_use_ffi.ipynb

```diff
@@ -0,0 +1,221 @@
whG6F9AMqlwSSEUs6h2pueUo4jAEWrKdQJU8ydbBZ7kplJbORQ47V9rFl4/7KX4fr7+/uv/v7aZLfzrx0XAgDaV1MA5aWHjx+EUWDQVwBxjOGW8zdgjub0Fj2+kRxoR639kl6JJdYsHqYnAdbSfwDQJbkKQijlHKo9VwFwhdA+hBRqAFPi4KPlfaNs4zlTSoNAXZw36zmGAECnSgvsQw41BvbPbd4J7s8heA9cUmv45CVvkgSA+MY3YI7mvK1SYAbq10rvMFft+y24zxRrgvojz1L6VPs1siSO5S/pP6BgLWcPWt43yiZXwRrOG4BkbnIXAM0IOYApbTDU8r5Rj9Np2h8omWsgAA2rPdgJ51IF9kNMcINYWrmut7IfADm1OBm4xX0CAADyaKW/aGU/COf93d1Xf9YS2AeAjrScPWh536iHXAUtcA0EGnabuwBoQozBwmZTxkC55X0D2rb2+nXpOrXms2u6BmqGAZhIIJKWpArsQ8lau65v3g3D6UvuKgDq1HJow5skAQAgrJb7h7e0ts/6pPblWkzYs5R+tXadzMmxBKrRcvag5X0D2iZXAZDUTe4CgAtyhyVzbx9gqaXXr6mrC65dhdD1FYCGtBbspG8C+9Dudb3V/QKIqYeJwD3sI23LFbgBAKDdfqLV/SKPz09PnqUAAGHknnebe/sAS8lV9MuCCJCN0D4AwDAsb0o0MwAATSg9sG9SGym0Hmxvff8AAK7RVwAAsfQW9G59f1vfP9LQf+BaEo5jCQBAUnIV9fO9gKyE9qF0uVYNsloRUKsl16+1TcmSr+/hOnu+cp7GD6BJwo+0ovTAPqTQyzW9l/0EWKunicA97SsAAMTQ25i6l/3tZT97MvdZyFKfn548R8E1JCDHEuCMXAXAPHIVfZLbgOyE9gGAduRoLNd8TukN5tx9EtIH6IrQI63IFdhPNTkOpujtmt7b/gLM1eNE4B73GQAAmK+33qG3/WW5MagvrA8AAFAIuYq0XuYocv4BshPahxDc1OZxvIBShL4etXh9u7RPGjyAbgk70opcgX0AgFIJZAAAAHPoIYBhsFAxX3NvCMexBKpkLuk8jhdQCrkKgGBucxcAzRgHFDFW99ls0g5YYq1QZNAFxDT32hXrmnQ6zasl9TV+idLrAwBYwAQy+FGvC7Fs3g3D6UvuKgD49N33V/9NqsnJDx8/TKoHSmERMgAgt96ChL3t70ivxFRjj6L36Fuv18oYHEuganIV15mXC8QkVwGQldA+hPbaACFWs1YyAyUAAGhSr+FOyDnJzAQ3WnEtJO8eA5BXqonAc8Me5//eZGUAAMjPuBzqk3Ih4/d3d55rdMi9IRzHEmiKXMWP5CoAALoitA+1sGIQQDixr6dzV4UDgEoIU9IKb6SEH6W4rs95m/35v01R2+bdvPoAWCfEmxnHz4g1edkbJAEA4LJeg4Qp93tKT5L6+6BXYq7xOYznK33o9d4Qg2MJMJFcBUA4chUAwQntA18zGAJqVPu1yy8QAaiAwD6tKCGwn/KNNpDL2jD8+PXuPwD1ixHs+PTd9yYxAwBAQj2Pv1Pt+5ze6fzf9vy9oXzv7+4E9xvm+hOOYwlQqNrnJgN9qv3aJVcBNEBoHwDoS6omzqpwAADFKSGwD6WIGYYP+fb605e4tW7eha0XoFaxJgbHfBNjrOC+N0hSAwuBQbncQ4DWCBHGt/bekWpRM71S3eY+7wjZcwjut8e9IRzHEgCArOQqAKIQ2oea5F4xyEAJKFEr16Xc13gAuMBbjmlBrYH9UuqAqWIE4MfPdD8CqEuKMEeqcArUTl8BaQgyAq0wxk4n1L1j/BzfO0I57yEsGsYwuL6E5FgCBJJ7zq1cBVCiVq5Lua/xACsJ7UMpSmjccm8fILbUzVsJ13YAWElAkhbUGtiHWGJd272xHoBRytCi4D5AHi0E1EPeP1o4HkCfjKUvi3l8Ytw7YvdHDx8/uOd1aHxmsia8//7uzrOXirg3hONYAixUwtzb3NsHiE2uAiAaoX0AAADgVQL7tEBgH9JIEdg/fXFvAoihhcnDgvsAzCWwD/TCOLlMMe8d+iNiWRveF9wvh2tEOI4lAAAA8NJN7gKAmWKtLHTtc1OvogQAAAArlRjYX/MmGiDO4gAWAgAIq4XgognXAG0T2Adgilh9gXsHtRO8BwCKIVcBAECBhPahJBq4Mmw2y/8Aac35uct1jZ2zXdcRAAoiwEjtSgzsQ6tiBOkBAHidhcBgHYF9AHrgHkVsnqkAANHJVZRBrgLqIVcBUAyhfaBvoRtEzebXNOcAAFUS2Kd2rYRYTHojBtd4AGLKGQwRSoHX6SvgZwL7AOSW8v7hXkVsS3qNVp7fAAA0Sa4iLrkKAOiG0D7UKPQg/NrntbZSXcpmprfmae3+9na8AACA4JZM+BJigeVOX/rYJgAAwEgIEoCehVwEBwAAVpOrWEeuIh65CgDoltA+lKa1Rq4kJTQsJdQQWsxmUKMJAJCcNzDTG4F9AAAAaJ+AIQBzxLhvWPSFFnnGAgBEJVcRTwnz80uoITS5CgBgENoHelBic1JiTXOl3ocWjhlhzTkfcv/ibs72necAZBQqsO8NyOTy/u5u1r9PPZlsbn0AAADAeiGDlwKXANTEfYsSeVYCAJBRifPxS6xpLrkKcpOrACiK0D7UKtTg49rn5B6QrVFDM1JDjS/lrjn39gEAGiWwT+1KD+xDi1zzAQDSEm6B+QT2AQDi8rwFAMhOruK6Gubf11DjS7lrzr19AOBVQvtQopobulLU1nzUUG9pTV1JtQAAMAyD8Cb5tBjYr6FG6hNqgRYAAOqgr4BwBPYBWMN9BAAAFpKrWK+2Ofc11CtXAQBccJu7AKBgNTa5NTccY+0lHvdSj2vJxwwAoCJCnPREaAUAAAD68PDxQ+4SAAAAAGhBjXPVS53/P0XJGYFSj2vJxwwAOiO0D7HFHJRvNusG1SXXtnSbLchx7C6p4biWdswAACoSKrB/+hLmc2Cu93
```

Comment:
The download happens when you do the .info command. Move this warning to that step?

Reply:
Moved the warning up earlier in the notebook as suggested.